crates/viewer/re_viewport/src/system_execution.rs

```diff
@@ -121,6 +121,10 @@ pub fn execute_systems_for_all_views<'a>(
     views: &'a BTreeMap<ViewId, ViewBlueprint>,
     view_states: &mut ViewStates,
 ) -> HashMap<ViewId, (ViewQuery<'a>, SystemExecutionOutput)> {
+    // TODO(andreas): wat? why is this not the ctx query?????
+    // Like so:
+    //let time_int = ctx.current_query().at();
+    //
```

Comment:
that's a bit too informal to go in 😅. Comment on `latest_at` is enough (and worth trying to fix in a follow-up)